docs/admin/kubeadm-alpha.md

```diff
@@ -0,0 +1,266 @@
+* [Kubeadm alpha phase etcd](#cmd-phase-etcd)
+* [Kubeadm alpha phase mark-master](#cmd-phase-mark-master)
+* [Kubeadm alpha phase bootstrap-token](#cmd-phase-bootstrap-token)
+* [Kubeadm alpha phase upload-config](#cmd-phase-upload-config)
+* [Kubeadm alpha phase addon](#cmd-phase-addon)
+* [Kubeadm alpha phase selfhosting](#cmd-phase-self-hosting)
+
+## Kubeadm alpha phase preflight {#cmd-phase-preflight}
+
+You can execute preflight checks both for master node, like in `kubeadm init`, or for worker node
+like in `kubeadm join`.
+
+{% capture preflight_master %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_preflight_master.md %}
+{% endcapture %}
+
+{% capture preflight_node %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_preflight_node.md %}
+{% endcapture %}
+
+{% assign tab_set_name = "tab-preflight" %}
+{% assign tab_names = "master,node" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: preflight_master | push: preflight_node %}
+
+{% include tabs.md %}
+
+## Kubeadm token create {#cmd-phase-token}
+
+Currently there is no a specific command under `kubeadm alpha phase` for creating a token; however
+you can use [kubeadm token create](/docs/admin/kubeadm-token.md#cmd-token-create) command.
+
+## Kubeadm alpha phase certs {#cmd-phase-certs}
+
+You can create all required certificates with `all` sub command, or create selectively certificates.
+
+{% capture certs_all %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_certs_all.md %}
+{% endcapture %}
+
+{% capture certs_ca %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_certs_ca.md %}
+{% endcapture %}
+
+{% capture certs_apiserver %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_certs_apiserver.md %}
+{% endcapture %}
+
+{% capture certs_apiserver-kubelet-client %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_certs_apiserver-kubelet-client.md %}
+{% endcapture %}
+
+{% capture certs_sa %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_certs_sa.md %}
+{% endcapture %}
+
+{% capture certs_front-proxy-ca %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_certs_front-proxy-ca.md %}
+{% endcapture %}
+
+{% capture certs_front-proxy-client %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_certs_front-proxy-client.md %}
+{% endcapture %}
+
+{% assign tab_set_name = "tab-certs" %}
+{% assign tab_names = "all,ca,apiserver,apiserver-kubelet-client,sa,front-proxy-ca,front-proxy-client" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: certs_all | push: certs_ca | push: certs_apiserver | push: certs_apiserver-kubelet-client | push: certs_sa | push: certs_front-proxy-ca | push: certs_front-proxy-client %}
+
+{% include tabs.md %}
+
+## Kubeadm alpha phase kubeconfig {#cmd-phase-kubeconfig}
+
+You can create all required kubeconfig files with `all` sub command, or create selectively kubeconfig file; additionally, it is also available the `user` sub command supporting creation of kubeconfig files for additional users.
```

Comment:
with the `all` subcommand, or selectively create the file. Additionally, the `user` subcommand supports the creation of ...